[A,scripts/admin/kcfinder/core/uploader.php,57]         require ROOT . "/scripts/admin/kcfinder/config.php";
[C,scripts/admin/kcfinder/core/uploader.php,58] 
[C,scripts/admin/kcfinder/core/uploader.php,59]         // SETTING UP SESSION
[C,scripts/admin/kcfinder/core/uploader.php,60]         if (isset($_CONFIG['_sessionLifetime']))
@@ -66,7 +66,7 @@ public function __construct() {
[C,scripts/admin/kcfinder/core/uploader.php,66]         @session_start();
[C,scripts/admin/kcfinder/core/uploader.php,67] 
[C,scripts/admin/kcfinder/core/uploader.php,68]         // RELOAD DEFAULT CONFIGURATION
[D,scripts/admin/kcfinder/core/uploader.php,69]         require "config.php";
[A,scripts/admin/kcfinder/core/uploader.php,69]         require ROOT . "/scripts/admin/kcfinder/config.php";
[C,scripts/admin/kcfinder/core/uploader.php,70]         $this->config = $_CONFIG;
[C,scripts/admin/kcfinder/core/uploader.php,71] 
[C,scripts/admin/kcfinder/core/uploader.php,72]         // LOAD SESSION CONFIGURATION IF EXISTS
@@ -414,7 +414,7 @@ protected function makeThumb($file, $overwrite=true) {
[C,scripts/admin/kcfinder/core/uploader.php,414]     }
[C,scripts/admin/kcfinder/core/uploader.php,415] 
[C,scripts/admin/kcfinder/core/uploader.php,416]     protected function localize($langCode) {
[D,scripts/admin/kcfinder/core/uploader.php,417]         require "lang/{$langCode}.php";
[A,scripts/admin/kcfinder/core/uploader.php,417]         require ROOT . "/scripts/admin/kcfinder/lang/{$langCode}.php";
[C,scripts/admin/kcfinder/core/uploader.php,418]         setlocale(LC_ALL, $lang['_locale']);
[C,scripts/admin/kcfinder/core/uploader.php,419]         $this->charset = $lang['_charset'];
[C,scripts/admin/kcfinder/core/uploader.php,420]         $this->dateTimeFull = $lang['_dateTimeFull'];
@@ -1,245 +0,0 @@
[D,scripts/admin/kcfinder/css.php,1] <?php
[D,scripts/admin/kcfinder/css.php,2] 
[D,scripts/admin/kcfinder/css.php,3] /** This file is part of KCFinder project
[D,scripts/admin/kcfinder/css.php,4]   *
[D,scripts/admin/kcfinder/css.php,5]   *      @desc Base CSS definitions
[D,scripts/admin/kcfinder/css.php,6]   *   @package KCFinder
[D,scripts/admin/kcfinder/css.php,7]   *   @version 2.21
[D,scripts/admin/kcfinder/css.php,8]   *    @author Pavel Tzonkov <pavelc@users.sourceforge.net>
[D,scripts/admin/kcfinder/css.php,9]   * @copyright 2010 KCFinder Project
[D,scripts/admin/kcfinder/css.php,10]   *   @license http://www.opensource.org/licenses/gpl-2.0.php GPLv2
[D,scripts/admin/kcfinder/css.php,11]   *   @license http://www.opensource.org/licenses/lgpl-2.1.php LGPLv2
[D,scripts/admin/kcfinder/css.php,12]   *      @link http://kcfinder.sunhater.com
[D,scripts/admin/kcfinder/css.php,13]   */
[D,scripts/admin/kcfinder/css.php,14] 
[D,scripts/admin/kcfinder/css.php,15] require "core/autoload.php";
[D,scripts/admin/kcfinder/css.php,16] $mtime = @filemtime(__FILE__);
[D,scripts/admin/kcfinder/css.php,17] if ($mtime) httpCache::checkMTime($mtime);
[D,scripts/admin/kcfinder/css.php,18] $browser = new browser();
[D,scripts/admin/kcfinder/css.php,19] $config = $browser->config;
[D,scripts/admin/kcfinder/css.php,20] ob_start();
[D,scripts/admin/kcfinder/css.php,21] 
[D,scripts/admin/kcfinder/css.php,22] ?>
[D,scripts/admin/kcfinder/css.php,23] html, body {
[D,scripts/admin/kcfinder/css.php,24]     overflow: hidden;
[D,scripts/admin/kcfinder/css.php,25] }
[D,scripts/admin/kcfinder/css.php,26] 
[D,scripts/admin/kcfinder/css.php,27] body, form, th, td {
[D,scripts/admin/kcfinder/css.php,28]     margin: 0;
[D,scripts/admin/kcfinder/css.php,29]     padding: 0;
[D,scripts/admin/kcfinder/css.php,30] }
[D,scripts/admin/kcfinder/css.php,31] 
[D,scripts/admin/kcfinder/css.php,32] a {
[D,scripts/admin/kcfinder/css.php,33]     cursor:pointer;
[D,scripts/admin/kcfinder/css.php,34] }
[D,scripts/admin/kcfinder/css.php,35] 
[D,scripts/admin/kcfinder/css.php,36] * {
[D,scripts/admin/kcfinder/css.php,37]     font-family: Tahoma, Verdana, Arial, sans-serif;
[D,scripts/admin/kcfinder/css.php,38]     font-size: 11px;
[D,scripts/admin/kcfinder/css.php,39] }
[D,scripts/admin/kcfinder/css.php,40] 
[D,scripts/admin/kcfinder/css.php,41] table {
[D,scripts/admin/kcfinder/css.php,42]     border-collapse: collapse;
[D,scripts/admin/kcfinder/css.php,43] }
[D,scripts/admin/kcfinder/css.php,44] 
[D,scripts/admin/kcfinder/css.php,45] #all {
[D,scripts/admin/kcfinder/css.php,46]     vvisibility: hidden;
[D,scripts/admin/kcfinder/css.php,47] }
[D,scripts/admin/kcfinder/css.php,48] 
[D,scripts/admin/kcfinder/css.php,49] #left {
[D,scripts/admin/kcfinder/css.php,50]     float: left;
[D,scripts/admin/kcfinder/css.php,51]     display: block;
[D,scripts/admin/kcfinder/css.php,52]     width: 25%;
[D,scripts/admin/kcfinder/css.php,53] }
[D,scripts/admin/kcfinder/css.php,54] 
[D,scripts/admin/kcfinder/css.php,55] #right {
[D,scripts/admin/kcfinder/css.php,56]     float: left;
[D,scripts/admin/kcfinder/css.php,57]     display: block;
[D,scripts/admin/kcfinder/css.php,58]     width: 75%;
[D,scripts/admin/kcfinder/css.php,59] }
[D,scripts/admin/kcfinder/css.php,60] 
[D,scripts/admin/kcfinder/css.php,61] #settings {
[D,scripts/admin/kcfinder/css.php,62]     display: none;
[D,scripts/admin/kcfinder/css.php,63]     padding: 0;
[D,scripts/admin/kcfinder/css.php,64]     float: left;
[D,scripts/admin/kcfinder/css.php,65]     width: 100%;
[D,scripts/admin/kcfinder/css.php,66] }
[D,scripts/admin/kcfinder/css.php,67] 
[D,scripts/admin/kcfinder/css.php,68] #settings > div {
[D,scripts/admin/kcfinder/css.php,69]     float: left;
[D,scripts/admin/kcfinder/css.php,70] }
[D,scripts/admin/kcfinder/css.php,71] 
[D,scripts/admin/kcfinder/css.php,72] #folders {
[D,scripts/admin/kcfinder/css.php,73]     padding: 5px;
[D,scripts/admin/kcfinder/css.php,74]     overflow: auto;
[D,scripts/admin/kcfinder/css.php,75] }
[D,scripts/admin/kcfinder/css.php,76] 
[D,scripts/admin/kcfinder/css.php,77] #toolbar {
[D,scripts/admin/kcfinder/css.php,78]     padding: 5px;
[D,scripts/admin/kcfinder/css.php,79] }
[D,scripts/admin/kcfinder/css.php,80] 
[D,scripts/admin/kcfinder/css.php,81] #files {
[D,scripts/admin/kcfinder/css.php,82]     padding: 5px;
[D,scripts/admin/kcfinder/css.php,83]     overflow: auto;
[D,scripts/admin/kcfinder/css.php,84] }
[D,scripts/admin/kcfinder/css.php,85] 
[D,scripts/admin/kcfinder/css.php,86] #status {
[D,scripts/admin/kcfinder/css.php,87]     padding: 5px;
[D,scripts/admin/kcfinder/css.php,88]     float: left;
[D,scripts/admin/kcfinder/css.php,89]     overflow: hidden;
[D,scripts/admin/kcfinder/css.php,90] }
[D,scripts/admin/kcfinder/css.php,91] 
[D,scripts/admin/kcfinder/css.php,92] #fileinfo {
[D,scripts/admin/kcfinder/css.php,93]     float: left;
[D,scripts/admin/kcfinder/css.php,94] }
[D,scripts/admin/kcfinder/css.php,95] 
[D,scripts/admin/kcfinder/css.php,96] #clipboard div {
[D,scripts/admin/kcfinder/css.php,97]     width: 16px;
[D,scripts/admin/kcfinder/css.php,98]     height: 16px;
[D,scripts/admin/kcfinder/css.php,99] }
[D,scripts/admin/kcfinder/css.php,100] 
[D,scripts/admin/kcfinder/css.php,101] .folders {
[D,scripts/admin/kcfinder/css.php,102]     margin-left: 16px;
[D,scripts/admin/kcfinder/css.php,103] }
[D,scripts/admin/kcfinder/css.php,104] 
[D,scripts/admin/kcfinder/css.php,105] div.file {
[D,scripts/admin/kcfinder/css.php,106]     overflow-x: hidden;
[D,scripts/admin/kcfinder/css.php,107]     width: <?php echo $config['thumbWidth'] ?>px;
[D,scripts/admin/kcfinder/css.php,108]     float: left;
[D,scripts/admin/kcfinder/css.php,109]     text-align: center;
[D,scripts/admin/kcfinder/css.php,110]     cursor: default;
[D,scripts/admin/kcfinder/css.php,111]     white-space: nowrap;
[D,scripts/admin/kcfinder/css.php,112] }
[D,scripts/admin/kcfinder/css.php,113] 
[D,scripts/admin/kcfinder/css.php,114] div.file .thumb {
[D,scripts/admin/kcfinder/css.php,115]     width: <?php echo $config['thumbWidth'] ?>px;
[D,scripts/admin/kcfinder/css.php,116]     height: <?php echo $config['thumbHeight'] ?>px;
[D,scripts/admin/kcfinder/css.php,117]     background: no-repeat center center;
[D,scripts/admin/kcfinder/css.php,118] }
[D,scripts/admin/kcfinder/css.php,119] 
[D,scripts/admin/kcfinder/css.php,120] #files table {
[D,scripts/admin/kcfinder/css.php,121]     width: 100%;
[D,scripts/admin/kcfinder/css.php,122] }
[D,scripts/admin/kcfinder/css.php,123] 
[D,scripts/admin/kcfinder/css.php,124] tr.file {
[D,scripts/admin/kcfinder/css.php,125]     cursor: default;
[D,scripts/admin/kcfinder/css.php,126] }
[D,scripts/admin/kcfinder/css.php,127] 
[D,scripts/admin/kcfinder/css.php,128] tr.file > td {
[D,scripts/admin/kcfinder/css.php,129]     white-space: nowrap;
[D,scripts/admin/kcfinder/css.php,130] }
[D,scripts/admin/kcfinder/css.php,131] 
[D,scripts/admin/kcfinder/css.php,132] tr.file > td.name {
[D,scripts/admin/kcfinder/css.php,133]     background-repeat: no-repeat;
[D,scripts/admin/kcfinder/css.php,134]     background-position: left center;
[D,scripts/admin/kcfinder/css.php,135]     padding-left: 20px;
[D,scripts/admin/kcfinder/css.php,136]     width: 100%;
[D,scripts/admin/kcfinder/css.php,137] }
[D,scripts/admin/kcfinder/css.php,138] 
[D,scripts/admin/kcfinder/css.php,139] tr.file > td.time,
[D,scripts/admin/kcfinder/css.php,140] tr.file > td.size {
[D,scripts/admin/kcfinder/css.php,141]     text-align: right;
[D,scripts/admin/kcfinder/css.php,142] }
[D,scripts/admin/kcfinder/css.php,143] 
[D,scripts/admin/kcfinder/css.php,144] #toolbar {
[D,scripts/admin/kcfinder/css.php,145]     cursor: default;
[D,scripts/admin/kcfinder/css.php,146]     white-space: nowrap;
[D,scripts/admin/kcfinder/css.php,147] }
[D,scripts/admin/kcfinder/css.php,148] 
[D,scripts/admin/kcfinder/css.php,149] #toolbar a {
[D,scripts/admin/kcfinder/css.php,150]     padding-left: 20px;
[D,scripts/admin/kcfinder/css.php,151]     text-decoration: none;
[D,scripts/admin/kcfinder/css.php,152]     background: no-repeat left center;
[D,scripts/admin/kcfinder/css.php,153] }
[D,scripts/admin/kcfinder/css.php,154] 
[D,scripts/admin/kcfinder/css.php,155] #toolbar a:hover, a[href="#upload"].uploadHover {
[D,scripts/admin/kcfinder/css.php,156]     color: #000;
[D,scripts/admin/kcfinder/css.php,157] }
[D,scripts/admin/kcfinder/css.php,158] 
[D,scripts/admin/kcfinder/css.php,159] #upload {
[D,scripts/admin/kcfinder/css.php,160]     position: absolute;
[D,scripts/admin/kcfinder/css.php,161]     overflow: hidden;
[D,scripts/admin/kcfinder/css.php,162]     opacity: 0;
[D,scripts/admin/kcfinder/css.php,163]     filter: alpha(opacity:0);
[D,scripts/admin/kcfinder/css.php,164] }
[D,scripts/admin/kcfinder/css.php,165] 
[D,scripts/admin/kcfinder/css.php,166] #upload input {
[D,scripts/admin/kcfinder/css.php,167]     cursor: pointer;
[D,scripts/admin/kcfinder/css.php,168] }
[D,scripts/admin/kcfinder/css.php,169] 
[D,scripts/admin/kcfinder/css.php,170] #uploadResponse {
[D,scripts/admin/kcfinder/css.php,171]     display: none;
[D,scripts/admin/kcfinder/css.php,172] }
[D,scripts/admin/kcfinder/css.php,173] 
[D,scripts/admin/kcfinder/css.php,174] span.brace {
[D,scripts/admin/kcfinder/css.php,175]     padding-left: 11px;
[D,scripts/admin/kcfinder/css.php,176]     cursor: default;
[D,scripts/admin/kcfinder/css.php,177] }
[D,scripts/admin/kcfinder/css.php,178] 
[D,scripts/admin/kcfinder/css.php,179] span.brace.opened, span.brace.closed {
[D,scripts/admin/kcfinder/css.php,180]     cursor: pointer;
[D,scripts/admin/kcfinder/css.php,181] }
[D,scripts/admin/kcfinder/css.php,182] 
[D,scripts/admin/kcfinder/css.php,183] #shadow {
[D,scripts/admin/kcfinder/css.php,184]     position: absolute;
[D,scripts/admin/kcfinder/css.php,185]     top: 0;
[D,scripts/admin/kcfinder/css.php,186]     left: 0;
[D,scripts/admin/kcfinder/css.php,187]     display: none;
[D,scripts/admin/kcfinder/css.php,188]     background: #fff;
[D,scripts/admin/kcfinder/css.php,189]     z-index: 100;
[D,scripts/admin/kcfinder/css.php,190]     opacity: 0.5;
[D,scripts/admin/kcfinder/css.php,191]     filter: alpha(opacity:50);
[D,scripts/admin/kcfinder/css.php,192] }
[D,scripts/admin/kcfinder/css.php,193] 
[D,scripts/admin/kcfinder/css.php,194] #dialog, #clipboard {
[D,scripts/admin/kcfinder/css.php,195]     position: absolute;
[D,scripts/admin/kcfinder/css.php,196]     display: none;
[D,scripts/admin/kcfinder/css.php,197]     z-index: 101;
[D,scripts/admin/kcfinder/css.php,198]     cursor: default;
[D,scripts/admin/kcfinder/css.php,199] }
[D,scripts/admin/kcfinder/css.php,200] 
[D,scripts/admin/kcfinder/css.php,201] #clipboard {
[D,scripts/admin/kcfinder/css.php,202]     z-index: 99;
[D,scripts/admin/kcfinder/css.php,203] }
[D,scripts/admin/kcfinder/css.php,204] 
[D,scripts/admin/kcfinder/css.php,205] #loading {
[D,scripts/admin/kcfinder/css.php,206]     display: none;
[D,scripts/admin/kcfinder/css.php,207]     float: right;
[D,scripts/admin/kcfinder/css.php,208] }
[D,scripts/admin/kcfinder/css.php,209] 
[D,scripts/admin/kcfinder/css.php,210] .menu {
[D,scripts/admin/kcfinder/css.php,211]     background: #888;
[D,scripts/admin/kcfinder/css.php,212]     white-space: nowrap;
[D,scripts/admin/kcfinder/css.php,213] }
[D,scripts/admin/kcfinder/css.php,214] 
[D,scripts/admin/kcfinder/css.php,215] .menu a {
[D,scripts/admin/kcfinder/css.php,216]     display: block;
[D,scripts/admin/kcfinder/css.php,217] }
[D,scripts/admin/kcfinder/css.php,218] 
[D,scripts/admin/kcfinder/css.php,219] .menu .list {
[D,scripts/admin/kcfinder/css.php,220]     max-height: 0;
[D,scripts/admin/kcfinder/css.php,221]     overflow-y: auto;
[D,scripts/admin/kcfinder/css.php,222]     overflow-x: hidden;
[D,scripts/admin/kcfinder/css.php,223]     white-space: nowrap;
[D,scripts/admin/kcfinder/css.php,224] }
[D,scripts/admin/kcfinder/css.php,225] 
[D,scripts/admin/kcfinder/css.php,226] .file .access, .file .hasThumb {
[D,scripts/admin/kcfinder/css.php,227]     display: none;
[D,scripts/admin/kcfinder/css.php,228] }
[D,scripts/admin/kcfinder/css.php,229] 
[D,scripts/admin/kcfinder/css.php,230] #dialog img {
[D,scripts/admin/kcfinder/css.php,231]     cursor: pointer;
[D,scripts/admin/kcfinder/css.php,232] }
[D,scripts/admin/kcfinder/css.php,233] 
[D,scripts/admin/kcfinder/css.php,234] #resizer {
[D,scripts/admin/kcfinder/css.php,235]     position: absolute;
[D,scripts/admin/kcfinder/css.php,236]     z-index: 98;
[D,scripts/admin/kcfinder/css.php,237]     top: 0;
[D,scripts/admin/kcfinder/css.php,238]     background: #000;
[D,scripts/admin/kcfinder/css.php,239]     opacity: 0;
[D,scripts/admin/kcfinder/css.php,240]     filter: alpha(opacity:0);
[D,scripts/admin/kcfinder/css.php,241] }
[D,scripts/admin/kcfinder/css.php,242] <?php
[D,scripts/admin/kcfinder/css.php,243] 
[D,scripts/admin/kcfinder/css.php,244] header("Content-Type: text/css");
[D,scripts/admin/kcfinder/css.php,245] echo text::compressCSS(ob_get_clean());
@@ -183,7 +183,7 @@ browser.humanSize = function(size) {
[C,scripts/admin/kcfinder/js/browser/misc.js,183] };
[C,scripts/admin/kcfinder/js/browser/misc.js,184] 
[C,scripts/admin/kcfinder/js/browser/misc.js,185] browser.baseGetData = function(act) {
[D,scripts/admin/kcfinder/js/browser/misc.js,186]     var data = 'browse.php?type=' + encodeURIComponent(this.type) + '&lng=' + this.lang;
[A,scripts/admin/kcfinder/js/browser/misc.js,186]     var data = 'browse?type=' + encodeURIComponent(this.type) + '&lng=' + this.lang;
[C,scripts/admin/kcfinder/js/browser/misc.js,187]     if (act)
[C,scripts/admin/kcfinder/js/browser/misc.js,188]         data += "&act=" + act
[C,scripts/admin/kcfinder/js/browser/misc.js,189]     return data;
@@ -1,33 +0,0 @@
[D,scripts/admin/kcfinder/js_localize.php,1] <?php
[D,scripts/admin/kcfinder/js_localize.php,2] 
[D,scripts/admin/kcfinder/js_localize.php,3] /** This file is part of KCFinder project
[D,scripts/admin/kcfinder/js_localize.php,4]   *
[D,scripts/admin/kcfinder/js_localize.php,5]   *      @desc Load language labels in JavaScript
[D,scripts/admin/kcfinder/js_localize.php,6]   *   @package KCFinder
[D,scripts/admin/kcfinder/js_localize.php,7]   *   @version 2.21
[D,scripts/admin/kcfinder/js_localize.php,8]   *    @author Pavel Tzonkov <pavelc@users.sourceforge.net>
[D,scripts/admin/kcfinder/js_localize.php,9]   * @copyright 2010 KCFinder Project
[D,scripts/admin/kcfinder/js_localize.php,10]   *   @license http://www.opensource.org/licenses/gpl-2.0.php GPLv2
[D,scripts/admin/kcfinder/js_localize.php,11]   *   @license http://www.opensource.org/licenses/lgpl-2.1.php LGPLv2
[D,scripts/admin/kcfinder/js_localize.php,12]   *      @link http://kcfinder.sunhater.com
[D,scripts/admin/kcfinder/js_localize.php,13]   */
[D,scripts/admin/kcfinder/js_localize.php,14] 
[D,scripts/admin/kcfinder/js_localize.php,15] require "core/autoload.php";
[D,scripts/admin/kcfinder/js_localize.php,16] 
[D,scripts/admin/kcfinder/js_localize.php,17] if (function_exists('set_magic_quotes_runtime'))
[D,scripts/admin/kcfinder/js_localize.php,18]     @set_magic_quotes_runtime(false);
[D,scripts/admin/kcfinder/js_localize.php,19] 
[D,scripts/admin/kcfinder/js_localize.php,20] $input = new input();
[D,scripts/admin/kcfinder/js_localize.php,21] if (!isset($input->get['lng']) || ($input->get['lng'] == 'en')) die;
[D,scripts/admin/kcfinder/js_localize.php,22] $file = "lang/" . $input->get['lng'] . ".php";
[D,scripts/admin/kcfinder/js_localize.php,23] $files = glob("lang/*.php");
[D,scripts/admin/kcfinder/js_localize.php,24] if (!in_array($file, $files)) die;
[D,scripts/admin/kcfinder/js_localize.php,25] $mtime = @filemtime($file);
[D,scripts/admin/kcfinder/js_localize.php,26] if ($mtime) httpCache::checkMTime($mtime);
[D,scripts/admin/kcfinder/js_localize.php,27] require $file;
[D,scripts/admin/kcfinder/js_localize.php,28] header("Content-Type: text/javascript; charset={$lang['_charset']}");
[D,scripts/admin/kcfinder/js_localize.php,29] foreach ($lang as $english => $native)
[D,scripts/admin/kcfinder/js_localize.php,30]     if (substr($english, 0, 1) != "_")
[D,scripts/admin/kcfinder/js_localize.php,31]         echo "browser.labels['" . text::jsValue($english) . "']=\"" . text::jsValue($native) . "\";";
[D,scripts/admin/kcfinder/js_localize.php,32] 
[D,scripts/admin/kcfinder/js_localize.php,33] ?>
@@ -1,2 +1,2 @@
[D,scripts/admin/kcfinder/tpl/tpl__css.php,1] <link href="css.php" rel="stylesheet" type="text/css" />
[D,scripts/admin/kcfinder/tpl/tpl__css.php,2] <link href="themes/<?php echo $this->config['theme'] ?>/style.css" rel="stylesheet" type="text/css" />
[A,scripts/admin/kcfinder/tpl/tpl__css.php,1] <link href="<?php echo Yii::app()->createUrl('admin/kcfinder/css'); ?>" rel="stylesheet" type="text/css" />
[A,scripts/admin/kcfinder/tpl/tpl__css.php,2] <link href="<?php echo Yii::app()->createUrl("scripts/admin/kcfinder/themes/{$this->config['theme']}/style.css"); ?>" rel="stylesheet" type="text/css" />
@@ -1,14 +1,14 @@
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,1] <script src="js/jquery.js" type="text/javascript"></script>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,2] <script src="js/jquery.rightClick.js" type="text/javascript"></script>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,3] <script src="js/jquery.drag.js" type="text/javascript"></script>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,4] <script src="js/helper.js" type="text/javascript"></script>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,5] <script src="js/browser/joiner.php" type="text/javascript"></script>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,6] <script src="js_localize.php?lng=<?php echo $this->lang ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,1] <script src="<?php echo Yii::app()->createUrl('scripts/admin/kcfinder/js/jquery.js'); ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,2] <script src="<?php echo Yii::app()->createUrl('scripts/admin/kcfinder/js/jquery.rightClick.js'); ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,3] <script src="<?php echo Yii::app()->createUrl('scripts/admin/kcfinder/js/jquery.drag.js'); ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,4] <script src="<?php echo Yii::app()->createUrl('scripts/admin/kcfinder/js/helper.js'); ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,5] <script src="<?php echo Yii::app()->createUrl('scripts/admin/kcfinder/js/browser/joiner.php'); ?>" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,6] <script src="<?php echo Yii::app()->createUrl("admin/kcfinder/js_localize?lng={$this->lang}"); ?>" type="text/javascript"></script>
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,7] <?php IF (isset($this->opener['TinyMCE']) && $this->opener['TinyMCE']): ?>
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,8] <script src="<?php echo $this->config['_tinyMCEPath'] ?>/tiny_mce_popup.js" type="text/javascript"></script>
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,9] <?php ENDIF ?>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,10] <?php IF (file_exists("themes/{$this->config['theme']}/init.js")): ?>
[D,scripts/admin/kcfinder/tpl/tpl__javascript.php,11] <script src="themes/<?php echo $this->config['theme'] ?>/init.js" type="text/javascript"></script>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,10] <?php IF (file_exists(ROOT . "/scripts/admin/kcfinder/themes/{$this->config['theme']}/init.js")): ?>
[A,scripts/admin/kcfinder/tpl/tpl__javascript.php,11] <script src="<?php echo Yii::app()->createUrl("scripts/admin/kcfinder/themes/{$this->config['theme']}/init.js"); ?>" type="text/javascript"></script>
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,12] <?php ENDIF ?>
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,13] <script type="text/javascript">
[C,scripts/admin/kcfinder/tpl/tpl__javascript.php,14] browser.support.chromeFrame = <?php echo (strpos(strtolower($_SERVER['HTTP_USER_AGENT']), " chromeframe") !== false) ? "true" : "false" ?>;
@@ -2,8 +2,8 @@
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,2] <html xmlns="http://www.w3.org/1999/xhtml">
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,3] <head>
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,4] <title>KCFinder: /<?php echo $this->session['dir'] ?></title>
[D,scripts/admin/kcfinder/tpl/tpl_browser.php,5] <?php INCLUDE "tpl/tpl__css.php" ?>
[D,scripts/admin/kcfinder/tpl/tpl_browser.php,6] <?php INCLUDE "tpl/tpl__javascript.php" ?>
[A,scripts/admin/kcfinder/tpl/tpl_browser.php,5] <?php INCLUDE ROOT . "/scripts/admin/kcfinder/tpl/tpl__css.php" ?>
[A,scripts/admin/kcfinder/tpl/tpl_browser.php,6] <?php INCLUDE ROOT . "/scripts/admin/kcfinder/tpl/tpl__javascript.php" ?>
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,7] </head>
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,8] <body>
[C,scripts/admin/kcfinder/tpl/tpl_browser.php,9] <script type="text/javascript">